src/sagemaker/cli/compatibility/v2/modifiers/tf_legacy_mode.py

```diff
@@ -148,3 +147,38 @@ def _to_ast_keyword(self, hps):
             return ast.keyword(arg="hyperparameters", value=ast.Dict(keys=keys, values=values))
 
         return None
+
+
+class TensorBoardParameterRemover(Modifier):
+    """A class for removing the ``run_tensorboard_locally`` parameter from ``fit()``."""
+
+    def node_should_be_modified(self, node):
+        """Checks if the ``ast.Call`` node invokes a function named "fit" and
+        contains a keyword argument named "run_tensorboard_locally".
+
+        Args:
+            node (ast.Call): a node that represents a function call. For more,
+                see https://docs.python.org/3/library/ast.html#abstract-grammar.
+
+        Returns:
+            bool: If the ``ast.Call`` is invoking a function named "fit" with
+                a parameter named "run_tensorboard_locally".
+        """
+        is_fit_call = isinstance(node.func, ast.Attribute) and node.func.attr == "fit"
+        if is_fit_call:
+            for kw in node.keywords:
+                if kw.arg == "run_tensorboard_locally":
```

Comment:
I think this can be expressed more concisely as
```
if "run_tensorboard_locally" in node.keywords:
```

But either way is fine. No action needed.